content/Api/Blip.Api.Template/Blip.Api.Template.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <ProjectGuid>sonarqube-guid-ui</ProjectGuid>
+    <ProjectGuid>{2E3DD875-4CFB-41EF-B627-7BA155C63F93}</ProjectGuid>
```

Comment:
whenever you build the project, or open in VS even, the placeholder gets replaced by the generated GUID. Revert back to the placeholder